// @ikas/storefront-assets/checkout/de.js
module.exports = {
  actions: {
    login: "Einloggen",
    signup: "Jetzt registrieren",
    edit: "Bearbeiten",
    apply: "Anwenden",
    close: "Schießen",
    cancel: "Abbrechen",
    save: "Speichern",
    continue: "Weiter",
    show: "Anzeigen",
    hide: "Verstecken",
    sendMessage: "Nachricht senden",
    continueWithShipping: "Weiter mit Versand",
    continueWithPayment: "Weiter mit Zahlung",
    completeOrder: "Bestellung abschließen",
    backToShopping: "Bestellung abschließen",
    backToCart: "Zurück zum Warenkorb",
    addToCart: "Zum Warenkorb hinzufügen",
    updateCart: "Meinen Warenkorb Aktualisieren",
  },
  securePayment: "Sichere Bezahlung",
  alreadyHaveAnAccount: "Du hast bereits ein Konto?",

  address: "Adresse",
  addressError: "Adresse eingeben",
  missingAddressFields: "Es fehlen einige Pflichtfelder in dieser Adresse.",
  editAddress: "Adresse bearbeiten",
  deleteAddress: "Adresse löschen",
  saveAddressCta: "Speichere diese Adresse für zukünftige Einkäufe",
  firstName: "Vorname",
  firstNameError: "Vorname eingeben",
  lastName: "Nachname",
  lastNameError: "Nachname eingeben",
  identityNumber: "Personalausweisnummer",
  identityNumberError: "Personalausweisnummer eingeben",
  addressTitle: "Name der Anschrift",
  adressTitleError: "Name für die Anschrift eingeben",
  addressLine1: "Addresse",
  addressLine2: "Adresszusatz",
  postalCode: "Postleitzahl",
  postalCodeError: "Postleitzahl eingeben",
  country: "Land",
  countryError: "Land auswählen",
  state: "Bundesland",
  stateError: "Bundesland eingeben",
  city: "Stadt",
  cityError: "Stadt eingeben",
  district: "Bezirk",
  districtError: "Bezirk eingeben",
  phone: "Telefonnummer",
  phoneError: "Eine gültige Telefonnummer eingeben",
  companyName: "Name des Unternehmens",
  taxOffice: "Finanzamt",
  taxNumber: "Umsatzsteuer-ID",
  password: "Passwort",
  passwordRequiredError: "Passwort eingeben",
  passwordMinError: "Muss mindestens {{value}} Zeichen lang sein!",

  deliveryMethod: "Zustellungsmethode",
  deliveryToAddress: "Lieferanschrift",
  inStoreDelivery: "Abholung",
  stockLocation: {
    deliveryTime: {
      WITHIN_ONE_HOUR: "In der Regel in 1 Stunde bereit",
      WITHIN_TWO_HOURS: "In der Regel in 2 Stunden bereit",
      WITHIN_FOUR_HOURS: "In der Regel in 4 Stunden bereit",
      WITHIN_TWENTY_FOUR_HOURS: "In der Regel innerhalb von 24 Stunden bereit",
      TWO_IN_FOUR_DAYS: "Gewöhnlich in 2-4 Tagen bereit",
      WITHIN_PLUS_FIVE_DAYS: "Gewöhnlich in +5 Tagen fertig",
    },
  },
  recipientsInfo: "Empfänger Details",
  pickUpLocation: "Abholort",
  receiver: "Empfänger",

  shipping: "Versand",
  payment: "Zahlung",
  summary: "Zusammenfassung",

  free: "Kostenlos",
  standartShipping: "Standardversand",
  returnPolicy: "Widerrufsrecht",
  privacyPolicy: "Datenschutzerklärung",
  termsOfService: "Geschäftsbedingungen",
  imprint: "Impressum",
  shippingPolicy: "Versandbedingungen",
  newAddress: "Neu Addresse",
  billingAddress: "Rechnungsaddresse",
  shippingAddress: "Versandaddresse",
  corporateInvoice: "Unternehmensrechnung",
  contactInfo: "Kontaktinformationen",
  email: "E-Mail",
  emailError: "Gebe eine gültige E-Mail ein",
  guestEmailTooltip:
    "Deine Bestellinformationen werden an diese E-Mail-Adresse gesendet",
  customerConsentCta: "Mich über Neuigkeiten und Angeboten informieren",
  useShippingForBilling: "Lieferadresse als Rechnungsaddresse verwenden",
  installmentOptions: "Ratenzahlungsoptionen",
  singleInstallment: "Einmalige Zahlung",
  installments: "Ratenzahlungen",
  creditCard: "Kreditkarte",

  ackPrivacyPolicy: "Datenschutzerklärung",
  ackTermsOfService: "Geschäftsbedingungen",
  ackPoliciesSentence:
    "Ich akzeptiere die {{ ackPrivacyPolicy }} und {{ ackTermsOfService }}.",

  securePaymentTooltip: "Alle Zahlungen sind sicher und verschlüsselt",
  giftPackage: "Geschenkpaket",
  giftPackageCta: "Diese Bestellung als Geschenk verpacken",
  giftPackageCtaPlaceholder: "Geschenknachricht eingeben",

  paymentMethod: "Zahlungsmethode",
  paymentMethodDiscount: "Rabatt",
  paymentMethodAdditionalPrice: "Gebühr",

  orderSuccessTitle: "Vielen Dank für Deine Bestellung!",
  orderSuccessDescription:
    "Hallo {{ customerName }}, wir haben Deine Bestellung erhalten. Wir benachrichtigen Dich, wenn die Bestellung versandt wird.",
  orderSuccessInStoreDeliveryDescription:
    "Hallo {{ customerName }}, wir haben Deine Bestellung erhalten. Wir werden Dich per E-Mail benachrichtigen, wenn die Bestellung in der Filiale eingetroffen ist.",
  orderNoAndDate: "Bestellnummer/Datum",
  paymentSummary: "Zahlungsübersicht",
  shippingSummary: "Versand Übersicht",
  helpCtaQuestion: "Brauchst Du Hilfe?",
  helpCta: "Kontaktiere uns",

  contactForm: "Kontakt-Formular",
  contactFormSent: "Nachricht erfolgreich versandt",
  contactFormDescription:
    "Wir werden so schnell wie möglich auf Deine Nachricht antworten.",
  contactFormCta: "Gebe Deine Nachricht ein",

  orderPackageStatus: {
    unfulfilled: "Nicht ausgeführt",
    readyForShipment: "Versandbereit",
    fulfilled: "Verschickt",
    partiallyFulfilled: "Teilweise verschickt",
    delivered: "Zugestellt",
    unableToDeliver: "Zugestellt fehlgeschlagen",
    cancelled: "Storniert",
    cancelRejected: "Storno abgeleht",
    cancelRequested: "Storno angefordert",
    refunded: "Rückerstattet",
    refundRequestAccepted: "Rückerstattungsanfrage angenommen",
    refundRejected: "Rückerstattungsanfrage abgeleht",
    refundRequested: "Rückerstattung angefragt",
  },

  quantity: "Menge",
  cartItemProduct: "Produkt",
  giftCodeErrorTitle: "Gutschein-/Rabattcode Fehler",
  giftCodeErrorDescription:
    "Der eingegebene Geschenkcode ist nicht gültig oder nicht auf die Produkte in Deinem Warenkorb anwendbar. Bitte überprüfe den Code und versuche es erneut.",
  mustBeSignInToApplyCampaign:
    "Du benötigst ein Konto, um die Vorteile der Kampagne nutzen zu können.",
  subtotal: "Zwischensumme",
  total: "Bestellsumme",
  subtotalTooltip:
    "Die Zwischensumme entspricht der Bestellsumme Deines Warenkorbs vor den Rabatten und beinhaltet nicht die Versandkosten.",
  cartShippingTitle: "Lieferung/Versand",
  cartTaxTitle: "MwSt.",
  cartInterest: "Zinsen",
  couponCode: "Gutschein-/Rabattcode",
  addCouponCode: "Gutschein-/Rabattcode hinzufügen",
  addOrderNote: "Bestellnotiz hinzufügen",
  editOrderNote: "Bestellnotiz bearbeiten",
  orderNote: "Bestellnotiz",
  enterPhoneNumber: "Telefonnummer eingeben",

  cardNumber: "Kartennummer",
  cardNumberError: "Bitte gebe eine gültige Kartennummer ein",
  cardHolderName: "Name des Karteninhabers",
  cardHolderError: "Name des Karteninhabers eingeben",
  cardExpiry: "Monat/Jahr",
  cardExpiryError: "Gebe ein gültiges Datum ein",
  cardCVC: "CVC",
  cardCVCError: "Gebe einen gültigen CVC ein",
  cardCVCTooltip: "CVC ist die 3-stellige Nummer hinter der Karte",

  masterPass: {
    paymentGatewaySelectBox: {
      delete: "Löschen",
    },
    creditCardForm: {
      checkboxLabel:
        "{{ link }} meine Karte in der {{ masterpassLogo }} Infrastruktur speichern.",
      checkboxLabelLink: "Ich möchte",
      moreInformation: "Mehr Informationen",
      cardName: "Name der Karte",
      phoneNumber: "Telefonnummer",
      button: "Verifizierungscode senden",
    },
    linkCardToClient: {
      text: "Du hast eine bei Masterpass registrierte Karte. Klicke auf OK, um diese Karte für Deinen Einkauf zu verwenden.",
      buttonText: "OK",
      subInfoText: "Zu diesem Zeitpunkt wird noch keine Zahlung durchgeführt.",
    },
    otpModal: {
      otpTextBank:
        "Gebe den SMS-Verifizierungscode ein, der dem Karteninhaber zugesandt wurde.",
      otpTextMasterPass:
        "Gebe den von Masterpass gesendeten SMS-Verifizierungscode ein.",
      countdownText: "Verbleibende Zeit: ",
      resendText: "Code erneut senden",
      inputLabel: "Gebe den Verifizierungscode ein",
      buttonText: "Verifizieren",
    },
    successModal: {
      otpSuccessText: "Deine Karte wurde erfolgreich registriert.",
      linkCardToClientSuccessText:
        "Dein Masterpass-Konto wurde erfolgreich bestätigt.",
    },
  },

  quickRegistration: {
    title: "Konto erstellen",
    subtitle:
      "Ab sofort kannst du deine Bestellungen als Kunde ganz einfach verfolgen und bleibst von aktuellen Kampagnen stets informiert.",
    completedTitle: "Dein Konto wurde erfolgreich erstellt!",
    completedSubtitle:
      "Ab sofort kannst du deine Bestellungen ganz einfach verfolgen und bleibst von aktuellen Kampagnen stets informiert.",
    accountEmail: "Deine E-Mail",
  },

  timeRemaining: "Verbleibende Zeit {{value}}",

  errorCustomerRequired:
    "Diese E-Mail-Adresse gehört einem registrierten Benutzer. Bitte logge Dich ein.",
  errorNoShipping: "Keine Versandoptionen für deine Region gefunden.",
  errorPayment:
    "Deine Zahlung konnte nicht empfangen werden. Bitte überprüfe Deine Angaben und versuche es erneut.",
  errorStockTitle: "Lagerbestand aktualisiert!",
  errorStockDescription:
    "Einige Artikel in Deinem Warenkorb sind nicht auf Lager. Wenn Du trotzdem fortfährst werden Produkte die nicht auf Lager sind aus Deinem Warenkorb entfernt?",
  errorUnknown:
    "Es ist ein unerwarteter Fehler aufgetreten. Bitte versuche es erneut.",
  errorPickupLocationRequired: "Abholort wählen",

  customizedProduct: "Individualisiertes Produkt",
  yes: "Ja",
  no: "Nein",
};
