// @ikas/storefront-assets/checkout/en.js
module.exports = {
  actions: {
    login: "Login",
    signup: "Sign Up",
    edit: "Edit",
    apply: "Apply",
    close: "Close",
    cancel: "Cancel",
    save: "Save",
    continue: "Continue",
    show: "Show",
    hide: "Hide",
    sendMessage: "Send Message",
    continueWithShipping: "Continue with Shipping",
    continueWithPayment: "Continue with Payment",
    completeOrder: "Complete the Order",
    backToShopping: "Back to Shopping",
    backToCart: "Back to Cart",
    addToCart: "Add to Cart",
    updateCart: "Update My Cart",
  },
  securePayment: "Secure Payment",
  alreadyHaveAnAccount: "Already have an account?",

  address: "Address",
  addressError: "Enter address",
  missingAddressFields:
    "There are some missing required fields in this address.",
  editAddress: "Edit Address",
  deleteAddress: "Delete Address",
  saveAddressCta: "Save this address for future purchases",
  firstName: "Name",
  firstNameError: "Enter name",
  lastName: "Surname",
  lastNameError: "Enter surname",
  identityNumber: "Identity Number",
  identityNumberError: "Enter identity number",
  addressTitle: "Adress Title",
  adressTitleError: "Enter address title",
  addressLine1: "Address",
  addressLine2: "Address Second Line",
  postalCode: "Postal Code",
  postalCodeError: "Enter postal code",
  country: "Country",
  countryError: "Select country",
  state: "State",
  stateError: "Select state",
  city: "City",
  cityError: "Select city",
  district: "District",
  districtError: "Select district",
  phone: "Phone",
  phoneError: "Enter a valid phone number",
  companyName: "Company Name",
  taxOffice: "Tax Office",
  taxNumber: "Tax Number",
  password: "Password",
  passwordRequiredError: "Enter password",
  passwordMinError: "Must be at least {{value}} characters!",

  deliveryMethod: "Delivery Method",
  deliveryToAddress: "Delivery to Address",
  inStoreDelivery: "In Store Delivery",
  stockLocation: {
    deliveryTime: {
      WITHIN_ONE_HOUR: "Usually ready in 1 hour",
      WITHIN_TWO_HOURS: "Usually ready in 2 hours",
      WITHIN_FOUR_HOURS: "Usually ready in 4 hours",
      WITHIN_TWENTY_FOUR_HOURS: "Usually ready in 24 hours",
      TWO_IN_FOUR_DAYS: "Usually ready in 2-4 days",
      WITHIN_PLUS_FIVE_DAYS: "Usually ready in +5 days",
    },
  },
  recipientsInfo: "Recipient Information",
  pickUpLocation: "Pick Up Location",
  receiver: "Receiver",

  shipping: "Shipping",
  payment: "Payment",
  summary: "Summary",

  free: "Free",
  standartShipping: "Standart Shipping",
  returnPolicy: "Return Policy",
  privacyPolicy: "Privacy Policy",
  termsOfService: "Terms of Service",
  imprint: "Imprint",
  shippingPolicy: "Shipping Policy",
  newAddress: "New Address",
  billingAddress: "Billing Address",
  shippingAddress: "Shipping Address",
  corporateInvoice: "Corporate invoice",
  contactInfo: "Contact Information",
  email: "E-Mail",
  emailError: "Enter a valid e-mail",
  guestEmailTooltip:
    "Your order information will be sent to this e-mail address",
  customerConsentCta: "Contact me about the news and promotions",
  useShippingForBilling: "Use shipping address for billing",
  installmentOptions: "Installment Options",
  singleInstallment: "Single Payment",
  installments: "Installments",
  creditCard: "Credit Card",

  ackPrivacyPolicy: "Privacy Policy",
  ackTermsOfService: "Terms of Service",
  ackPoliciesSentence:
    "I acknowledge the {{ ackPrivacyPolicy }} and {{ ackTermsOfService }}.",

  securePaymentTooltip: "All payments are secure and encrypted",
  giftPackage: "Gift Package",
  giftPackageCta: "Gift wrap this order",
  giftPackageCtaPlaceholder: "Enter your gift message",

  paymentMethod: "Payment Method",
  paymentMethodDiscount: "Discount",
  paymentMethodAdditionalPrice: "Fee",

  orderSuccessTitle: "Thank you for your order!",
  orderSuccessDescription:
    "Dear {{ customerName }}, we have received your order. We will notify you when your order is shipped.",
  orderSuccessInStoreDeliveryDescription:
    "Dear {{ customerName }}, we have received your order. We will notify you by e-mail when your order is delivered to the store.",
  orderNoAndDate: "Order Number / Date",
  paymentSummary: "Payment Summary",
  shippingSummary: "Shipping Summary",
  helpCtaQuestion: "Do you need help?",
  helpCta: "Contact us",

  contactForm: "Contact Form",
  contactFormSent: "Message sent successfully",
  contactFormDescription: "We will respond to your message as soon as possible",
  contactFormCta: "Type your message",

  orderPackageStatus: {
    unfulfilled: "Unfulfilled",
    readyForShipment: "Ready For Shipment",
    fulfilled: "Fulfilled",
    partiallyFulfilled: "Partially Fulfilled",
    delivered: "Delivered",
    unableToDeliver: "Unable to Deliver",
    cancelled: "Cancelled",
    cancelRejected: "Cancel Rejected",
    cancelRequested: "Cancel Requested",
    refunded: "Refunded",
    refundRequestAccepted: "Refund Request Accepted",
    refundRejected: "Refund Rejected",
    refundRequested: "Refund Requested",
  },

  quantity: "Quantity",
  cartItemProduct: "product",
  giftCodeErrorTitle: "Gift Code Error",
  giftCodeErrorDescription:
    "The gift code you entered is not valid or not applicable to the products in your cart. Please check your code and try again.",
  mustBeSignInToApplyCampaign:
    "You need to sign up to benefit from the discount.",
  subtotal: "Subtotal",
  total: "Total",
  subtotalTooltip:
    "Subtotal reflects the total for your cart before the discounts and does not include the shipping fees.",
  cartShippingTitle: "Delivery / Shipping",
  cartTaxTitle: "Tax",
  cartInterest: "Interest",
  couponCode: "Coupon Code",
  addCouponCode: "Add coupon code",
  addOrderNote: "Add order note",
  editOrderNote: "Edit order note",
  orderNote: "Order Note",
  enterPhoneNumber: "Enter phone number",

  cardNumber: "Card Number",
  cardNumberError: "Please enter a valid card number",
  cardHolderName: "Cardholder Name",
  cardHolderError: "Enter cardholder name",
  cardExpiry: "Month / Year",
  cardExpiryError: "Enter a valid date",
  cardCVC: "CVC",
  cardCVCError: "Enter a valid CVC",
  cardCVCTooltip: "CVC is the 3 digit number behind your card",

  masterPass: {
    paymentGatewaySelectBox: {
      delete: "Delete",
    },
    creditCardForm: {
      checkboxLabel:
        "I want to {{ link }} my card in the {{ masterpassLogo }} infrastructure.",
      checkboxLabelLink: "store",
      moreInformation: "More Information",
      cardName: "Card Name",
      phoneNumber: "Phone",
      button: "Send Verification Code",
    },
    linkCardToClient: {
      text: "You have a card registered to Masterpass. Click OK to use your card for your shopping.",
      buttonText: "OK",
      subInfoText: "Payment process will not be made at this stage.",
    },
    otpModal: {
      otpTextBank:
        "Enter the SMS verification code sent to the cardholder by your bank.",
      otpTextMasterPass: "Enter the SMS verification code sent by Masterpass.",
      countdownText: "Remaining time: ",
      resendText: "Resend code",
      inputLabel: "Enter the verification code",
      buttonText: "Verify",
    },
    successModal: {
      otpSuccessText: "Your card successfully registered.",
      linkCardToClientSuccessText:
        "Your Masterpass account has been successfully confirmed.",
    },
  },

  quickRegistration: {
    title: "Create Your Account",
    subtitle:
      "As a member, you can track your orders and get information about the latest offers and discounts.",
    completedTitle: "Account Created Successfully",
    completedSubtitle:
      "Now you can track your orders easily and keep up with the latest offers.",
    accountEmail: "Account email",
  },

  timeRemaining: "Time Remaining {{value}}",

  errorCustomerRequired:
    "This e-mail address belongs to a registered user. Please login to your account.",
  errorNoShipping: "Your region is not available for shipping.",
  errorPayment:
    "Your payment could not be received. Please check your information and try again.",
  errorStockTitle: "Stock Update!",
  errorStockDescription:
    "Some items in your cart are out of stock. Would you like to continue by removing these products from your cart?",
  errorUnknown: "An unexpected error has occurred. Please try again.",
  errorPickupLocationRequired: "Choose pickup location",

  customizedProduct: "Customized Product",
  yes: "Yes",
  no: "No",
};
