// @ikas/storefront-assets/checkout/tr.js
module.exports = {
  actions: {
    login: "Giriş Yap",
    signup: "Hesap Oluşturun",
    edit: "Düzenle",
    apply: "Uygula",
    close: "Kapat",
    cancel: "Vazgeç",
    save: "Kaydet",
    continue: "Devam Et",
    show: "Göster",
    hide: "Gizle",
    sendMessage: "Mesajı Gönder",
    continueWithShipping: "Kargo ile Devam Et",
    continueWithPayment: "Ödeme ile Devam Et",
    completeOrder: "Siparişi Tamamla",
    backToShopping: "Alışverişe Dön",
    backToCart: "Sepete Dön",
    addToCart: "Sepete Ekle",
    updateCart: "Sepetimi Güncelle",
  },
  securePayment: "Güvenli Ödeme",
  alreadyHaveAnAccount: "Zaten hesabınız var mı?",

  address: "Adres",
  addressError: "Adres girin",
  missingAddressFields: "Adres bilgilerinde bazı zorunlu alanlar eksik.",
  editAddress: "Adres Düzenle",
  deleteAddress: "Adresi Sil",
  saveAddressCta: "Bir sonraki işlem için bu adresi kaydet",
  firstName: "Ad",
  firstNameError: "Ad girin",
  lastName: "Soyad",
  lastNameError: "Soyad girin",
  identityNumber: "TC Kimlik No",
  identityNumberError: "TC kimlik no girin",
  addressTitle: "Adres Başlığı",
  adressTitleError: "Adres başlığı girin",
  addressLine1: "Adres",
  addressLine2: "Apartman, daire, vb.",
  postalCode: "Posta Kodu",
  postalCodeError: "Posta kodu girin",
  country: "Ülke",
  countryError: "Ülke seçin",
  state: "Eyalet",
  stateError: "Eyalet seçin",
  city: "İl",
  cityError: "İl seçin",
  district: "İlçe",
  districtError: "İlçe seçin",
  phone: "Telefon",
  phoneError: "Geçerli bir telefon girin",
  companyName: "Firma Adı",
  taxOffice: "Vergi Dairesi",
  taxNumber: "Vergi Numarası",
  password: "Şifre",
  passwordRequiredError: "Şifre girin",
  passwordMinError: "{{value}} karakterden büyük olmalı!",

  deliveryMethod: "Teslimat Yöntemi",
  deliveryToAddress: "Adrese Teslimat",
  inStoreDelivery: "Mağazadan Teslimat",
  stockLocation: {
    deliveryTime: {
      WITHIN_ONE_HOUR: "Genelde 1 saat içinde hazır olur",
      WITHIN_TWO_HOURS: "Genelde 2 saat içinde hazır olur",
      WITHIN_FOUR_HOURS: "Genelde 4 saat içinde hazır olur",
      WITHIN_TWENTY_FOUR_HOURS: "Genelde 24 saat içinde hazır olur",
      TWO_IN_FOUR_DAYS: "Genelde 2-4 gün içinde hazır olur",
      WITHIN_PLUS_FIVE_DAYS: "Genelde +5 gün içinde hazır olur",
    },
  },
  recipientsInfo: "Teslim Alacak Kişi Bilgileri",
  pickUpLocation: "Teslim Alınacak Konum",
  receiver: "Teslim Alacak Kişi",

  shipping: "Kargo",
  payment: "Ödeme",
  summary: "Özet",

  free: "Ücretsiz",
  standartShipping: "Standart Kargo",
  returnPolicy: "Para İade Politikası",
  privacyPolicy: "Gizlilik Politikası",
  termsOfService: "Hizmet Şartları",
  imprint: "Künye",
  shippingPolicy: "Nakliye Politikası",
  newAddress: "Yeni Adres",
  billingAddress: "Fatura Adresi",
  shippingAddress: "Teslimat Adresi",
  corporateInvoice: "Kurumsal fatura",
  contactInfo: "İletişim Bilgileri",
  email: "E-Posta",
  emailError: "Geçerli bir e-posta girin",
  guestEmailTooltip: "Sipariş bilgileri bu e-posta adresine gönderilecektir",
  customerConsentCta: "Beni haberlerden ve özel tekliflerden haberdar et",
  useShippingForBilling: "Fatura adresim teslimat adresimle aynı",
  installmentOptions: "Taksit Seçenekleri",
  singleInstallment: "Tek Çekim",
  installments: "Taksit",
  creditCard: "Kredi Kartı",

  ackPrivacyPolicy: "Gizlilik Sözleşmesini",
  ackTermsOfService: "Satış Sözleşmesini",
  ackPoliciesSentence:
    "{{ ackPrivacyPolicy }} ve {{ ackTermsOfService }} okudum, onaylıyorum.",

  securePaymentTooltip: "Ödemeler güvenli ve şifrelidir",
  giftPackage: "Hediye Paketi",
  giftPackageCta: "Hediye paketi istiyorum",
  giftPackageCtaPlaceholder: "Hediye mesajınızı giriniz",

  paymentMethod: "Ödeme Yöntemi",
  paymentMethodDiscount: "İndirim",
  paymentMethodAdditionalPrice: "İşlem Bedeli",

  orderSuccessTitle: "Siparişiniz için teşekkür ederiz!",
  orderSuccessDescription:
    "Sevgili {{ customerName }}, siparişiniz bize ulaşmıştır. Siparişiniz kargoya verildiğinde sizi e-posta ile bilgilendireceğiz.",
  orderSuccessInStoreDeliveryDescription:
    "Sevgili {{ customerName }}, siparişiniz bize ulaşmıştır. Siparişiniz mağazaya teslim edildiğinde sizi e-posta ile bilgilendireceğiz.",
  orderNoAndDate: "Sipariş No / Tarih",
  paymentSummary: "Ödeme Özeti",
  shippingSummary: "Teslimat Özeti",
  helpCtaQuestion: "Yardıma mı ihtiyacınız var?",
  helpCta: "Bizimle iletişime geçin",

  contactForm: "İletişim Formu",
  contactFormSent: "Mesaj başarıyla gönderildi",
  contactFormDescription:
    "Mesajınızla ilgili olarak size en kısa zamanda e-posta veya telefon aracılığıyla geri dönüş yapılacaktır.",
  contactFormCta: "Mesajınızı yazın",

  orderPackageStatus: {
    unfulfilled: "Oluşturuldu",
    readyForShipment: "Kargoya Hazır",
    fulfilled: "Gönderildi",
    partiallyFulfilled: "Kısmi Gönderildi",
    delivered: "Teslim Edildi",
    unableToDeliver: "Teslim Edilemedi",
    cancelled: "İptal Edildi",
    cancelRejected: "İptal Talebi Reddedildi",
    cancelRequested: "İptal İstendi",
    refunded: "İade Edildi",
    refundRequestAccepted: "İade Talebi Onaylandı",
    refundRejected: "İade Talebi Reddedildi",
    refundRequested: "İade Talep Edildi",
  },

  quantity: "Adet",
  cartItemProduct: "ürün",
  giftCodeErrorTitle: "İndirim Kodu Hatası",
  giftCodeErrorDescription:
    "İndirim kodunuz geçersiz veya sepetinizdeki ürünlere uygulanabilir değil. Lütfen kodunuzu kontrol edip tekrar deneyin.",
  mustBeSignInToApplyCampaign:
    "Kampanyadan faydalanabilmeniz için üye olmanız gerekmektedir.",
  subtotal: "Ara Toplam",
  total: "Toplam",
  subtotalTooltip:
    "Ara toplam, tüm geçerli indirimlerden önce siparişinizin toplam fiyatını yansıtır. kargo ücretini içermez.",
  cartShippingTitle: "Teslimat / Kargo",
  cartTaxTitle: "Vergi",
  cartInterest: "Vade Farkı",
  couponCode: "İndirim Kodu",
  addCouponCode: "İndirim kodu ekle",
  addOrderNote: "Sipariş notu ekle",
  editOrderNote: "Sipariş notu düzenle",
  orderNote: "Sipariş Notu",
  enterPhoneNumber: "Telefon numarası girin",

  cardNumber: "Kart Numarası",
  cardNumberError: "Geçerli bir kart numarası girin",
  cardHolderName: "Kart Üzerindeki İsim",
  cardHolderError: "Ad soyad girin",
  cardExpiry: "Ay / Yıl",
  cardExpiryError: "Geçerli bir tarih girin",
  cardCVC: "CVC",
  cardCVCError: "Geçerli bir güvenlik kodu girin",
  cardCVCTooltip: "Kartınızın arkasındaki 3 haneli güvenlik kodu",

  masterPass: {
    paymentGatewaySelectBox: {
      delete: "Sil",
    },
    creditCardForm: {
      checkboxLabel:
        "Kartımı {{ masterPassLogo }} altyapısında saklamak {{ link }}",
      checkboxLabelLink: "istiyorum.",
      moreInformation: "Detaylı Bilgi",
      cardName: "Kart İsmi",
      phoneNumber: "Telefon",
      button: "Onay Kodu Gönder",
    },
    linkCardToClient: {
      text: "MasterPass’e kayıtlı kartınız bulunmaktadır. Kartınızı alışverişinizde kullanmak için TAMAM’a tıklayınız.",
      buttonText: "Tamam",
      subInfoText: "Bu aşamada kartınızdan tahsilat yapılmayacaktır.",
    },
    otpModal: {
      otpTextBank:
        "Bankanız tarafından kart sahibine gönderilen doğrulama kodunu giriniz.",
      otpTextMasterPass:
        "Masterpass tarafından gönderilen SMS doğrulama kodunu giriniz.",
      countdownText: "Kalan Süre: ",
      resendText: "Yeni kod gönder",
      inputLabel: "Doğrulama Kodunu Giriniz",
      buttonText: "Doğrula",
    },
    successModal: {
      otpSuccessText: "Kartınız başarılı bir şekilde tanımlandı.",
      linkCardToClientSuccessText:
        "Masterpass hesabınız başarılı bir şekilde doğrulandı.",
    },
  },

  quickRegistration: {
    title: "Hesap Oluşturun",
    subtitle:
      "Üye olarak siparişinizi takip edebilir ve en güncel kampanyalardan haberdar olabilirsiniz.",
    completedTitle: "Hesabınız Başarıyla Oluşturuldu!",
    completedSubtitle:
      "Artık siparişinizi kolayca takip edebilir ve en güncel kampanyalardan haberdar olabilirsiniz.",
    accountEmail: "Üyelik e-postanız",
  },

  timeRemaining: "Kalan Süre {{value}}",

  errorCustomerRequired:
    "Bu e-posta adresine ait kayıtlı bir hesap bulunuyor. Lütfen hesabınıza giriş yapınız.",
  errorNoShipping: "Bulunduğunuz konuma gönderim yapılmamaktadır.",
  errorPayment:
    "Ödemeniz alınamadı. Lütfen bilgilerinizi kontrol edip tekrar deneyin.",
  errorStockTitle: "Stoklar Güncellendi!",
  errorStockDescription:
    "Sepetinizdeki bazı ürünlerin stoğu tükenmiştir. Bu ürünleri sepetinizden çıkararak devam etmek ister misiniz?",
  errorUnknown: "Beklenmedik bir hata oluştu. Lütfen tekrar deneyin.",
  errorPickupLocationRequired: "Teslim alınacak konumu seçin",

  customizedProduct: "Kişiselleştirilmiş Ürün",
  yes: "Evet",
  no: "Hayır",
};
